rpcserver.go

```diff
@@ -7193,7 +7193,12 @@ func (r *rpcServer) SendPayment(req *tchrpc.SendPaymentRequest,
 
 		update, err := updateStream.Recv()
 		if err != nil {
-			return err
+			// Stream is closed; no more updates.
+			if err == io.EOF {
```

Comment:
nit: Use `errors.Is()` instead? Though it shouldn't really be wrapped at this point, so doesn't matter too much.